mlir/test/Dialect/LLVMIR/genx-invalid.mlir

```diff
@@ -16,6 +16,22 @@ func.func @matrix_2Dblockload(%ptr : !llvm.ptr<i32>, %base_width : i32, %base_he
 
 // -----
 
+func.func @matrix_2Dblockstore(%ptr : !llvm.ptr<i32>, %base_width : i32, %base_height : i32, %base_pitch : i32, %x : i32, %y : i32, %stored_val : i32) {
+  // expected-error @+1 {{'genx.matrix.2Dblockstore' op invalid element size}}
```

Comment:
Is not clear what is the error. Which operand has invalid element size ?

Reply:
changed.